internal/core/src/common/BitsetView.cpp

```diff
@@ -0,0 +1,31 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "common/BitsetView.h"
+
+namespace milvus {
+
+BitsetView
+BitsetView::subview(size_t offset, size_t size) const {
```

Comment:
Since all implementations are coded in BitsetView.h, suggest to move subview to BitsetView.h

Reply:
sounds good. Prefer to do this in next pr.